### PR TITLE
Added enableComments as option to the maven build

### PR DESCRIPTION
**Pull request by boegemann**
I added the existing command line '--enable-comments' options as a configuration parameter to the Maven plugin. I left the default as false to keep behaviour.
To enable comments just set the enableComments property to true inside the plugin configuration:

      <plugin>
        <groupId>fr.inria.gforge.spoon</groupId>
        <artifactId>spoon-maven-plugin</artifactId>
        <configuration>
        	<enableComments>true</enableComments>
        </configuration>
      </plugin>

### CODE REVIEW

**Comment by surli** [comment]
Hi! Thanks for your PR, the feature is interesting! However it appears you used Windows-style end of lines in the file you changed, which messes up the diff in the PR. 
Could you put back linux-style end of lines please?

**Comment by boegemann** [comment]
Thanks for the feedback. Line endings should have changed now.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* text eol=crlf
```
Comment (surli) [inline .gitattributes]: It seems this line is useless.

Comment (boegemann) [inline .gitattributes]: The second line specifies the local method for line endings as opposed to the method used in the repository. Feel free to remove or to change to auto or ignore the file fully. Just used it to get the other files in line with your preferred style.

Comment (surli) [inline .gitattributes]: Yes I agree for the second one, but it overrides the first one, which is then useless... I'm not sure why you keep both in this file.

Comment (boegemann) [inline .gitattributes]: According to an Stack overflow the first one defines Server side and the second one client side.

Comment (boegemann) [inline .gitattributes]: https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings

Comment (surli) [inline .gitattributes]: I don't think that it's correct: it's specified in Git documentation: 
> When more than one pattern matches the path, a later line overrides an earlier line. This overriding is done per attribute. 

Source: https://git-scm.com/docs/gitattributes

Nevertheless I'll accept the PR and fix that after ;)

Comment (boegemann) [inline .gitattributes]: Cheers and thanks - great Project btw!

**Pull request merged**